lstchain/image/muon/muon_analysis.py

```diff
@@ -410,29 +427,38 @@ def create_muon_table():
 
 def fill_muon_event(output_parameters, good_ring, event_id, event_time,
                     muonintensityparam, muonringparam, radial_distribution,
-                    size_outside_ring, mean_pixel_charge_around_ring,
-                    hg_peak_sample=np.nan, lg_peak_sample=np.nan):
+                    size, size_outside_ring, mean_pixel_charge_around_ring,
+                    muonparameters, hg_peak_sample=np.nan,
+                    lg_peak_sample=np.nan):
 
     output_parameters['event_id'].append(event_id)
     output_parameters['event_time'].append(event_time)
-    output_parameters['ring_size'].append(muonintensityparam.ring_size)
+    output_parameters['ring_size'].append(size)
```

Comment:
let's keep in mind that the ring `size` (or better `intensity`) is missing from the ctapipe containers list. I'll open an issue